src/config/media/pipewire.md

```diff
@@ -0,0 +1,64 @@
+# PipeWire
+
+To use PipeWire, install the `pipewire` package.
+
+[pipewire(1)](https://man.voidlinux.org/pipewire) should be started as a user.
+Use the autostarting mechanism of your desktop environment or
+[startx](../graphical-session/xorg.md#startx).
+
+## PulseAudio replacement
+
+PipeWire can start a PulseAudio server. While it's possible to modify
+`/etc/pipewire/pipewire.conf` for auto-starting the PulseAudio server, it's
+recommended to start it from the same place where you start PipeWire by
+executing `pipewire-pulse`. This keeps PipeWire's configuration file unmodified
+for smoother future upgrades.
+
+After setting up `pipewire-pulse`, make sure that the PulseAudio service is
+[disabled](../services/index.md#disabling-services) and reboot.
+
+To check if the replacement is working correctly, use
+[pactl(1)](https://man.voidlinux.org/pactl) (provided by the `pulseaudio-utils`
+package):
+
+```
+$ pactl info
+
+...
+Server Name: PulseAudio (on PipeWire 0.3.18)
+...
+```
+
+##### Bluetooth audio
+
+For bluetooth audio to work, install the `libspa-bluetooth` package.
+
+## ALSA integration
+
+Install `alsa-pipewire`, then enable the PipeWire ALSA device and make it
+default:
+
+```
+# mkdir -p /etc/alsa/conf.d
+# ln -s /usr/share/alsa/alsa.conf.d/50-pipewire.conf /etc/alsa/conf.d
+# ln -s /usr/share/alsa/alsa.conf.d/99-pipewire-default.conf /etc/alsa/conf.d
+```
+
+## JACK replacement
+
+Install `libjack-pipewire`.
+
+Use [pw-jack(1)](https://man.voidlinux.org/pw-jack) to launch JACK clients
+manually:
+
+```
+$ pw-jack <application>
+```
+
+Alternatively, override the library provided by `libjack`. The following
+approach will work on `glibc`:
```

Comment:
Maybe link `ld.so(8)`?

Reply:
I prefer helping at least a bit. You can still copy from it, but you can also read to try and understand what is being done :)

Reply:
Right, i've added a link.